test/lib/log.spec.js

```diff
@@ -0,0 +1,52 @@
+"use strict";
+
+/* eslint-disable no-console */
+
+const { setLoggingOptions, error, log, warn } = require("../../lib/log");
+const chai = require("chai");
+const sinon = require("sinon");
+const sinonChai = require("sinon-chai");
+chai.use(sinonChai);
```

Comment:
All of this is already provided in https://github.com/FormidableLabs/trace-pkg/blob/main/test/setup.js and should not be repeated here.

Reply:
this has been fixed